db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
+      ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+      'BA61_tier1_budget_approver',
+      'ncr'
+    )
+    User.with_email_role_slug!(
+      ENV['NCR_BA61_TIER2_BUDGET_MAILBOX'] || 'communicart.ofm.approver@gmail.com',
+      'BA61_tier2_budget_approver',
+      'ncr'
+    )
+    User.with_email_role_slug!(
+      ENV['NCR_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+      'BA80_budget_approver',
+      'ncr'
+    )
+    User.with_email_role_slug!(
+      ENV['NCR_OOL_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+      'OOL_BA80_budget_approver',
+      'ncr'
+    )
+  end
+end
```

Comment:
yes. we need to migrate existing data (dev, stage, prod) and also provide base fixtures for a clean checkout.

If `db:seed` were part of a regular deploy, we wouldn't need both. But afaik, we never seed an existing, working db.